app/controllers/users_controller.rb

```diff
@@ -6,14 +6,14 @@ def index
     @users = policy_scope(User).order('LOWER(first_name), LOWER(last_name)')
   end
 
-  def show
-    @user = authorize User.find_by(id: params[:id])
-  end
-
   def new
     @user = authorize User.new
   end
 
+  def edit
+    @user = authorize User.find_by(id: params[:id])
```

Comment:
Szerintem érdemes ezt a sort kiszervezni egy before_action-be. Ugye a mostani PR-ban lett kódduplikáció. És elképzelhető, hogy a jövőben lesz majd destroy-unk vagy visszatér a show, csak másik feladattal, ahol megintcsak ezzel a sorral indítanánk feltételezhetőleg az akciót.